cypress/e2e/backoffice/company-role/company-role-create.cy.ts

```diff
@@ -0,0 +1,30 @@
+import { container } from '@utils';
+import { CompanyRoleCreateStaticFixtures } from '@interfaces/backoffice';
+import { CompanyRoleListPage, CompanyRoleCreatePage } from '@pages/backoffice';
+import { UserLoginScenario } from '@scenarios/backoffice';
+
+describe('company role create', { tags: ['@backoffice', '@company-role'] }, (): void => {
+  const companyRoleListPage = container.get(CompanyRoleListPage);
+  const companyRoleCreatePage = container.get(CompanyRoleCreatePage);
+  const userLoginScenario = container.get(UserLoginScenario);
+
+  let staticFixtures: CompanyRoleCreateStaticFixtures;
+
+  before((): void => {
+    staticFixtures = Cypress.env('staticFixtures');
+  });
+
+  beforeEach((): void => {
+    userLoginScenario.execute({
+      username: staticFixtures.rootUser.username,
+      password: staticFixtures.defaultPassword,
+    });
+  });
+
+  it('should be able to access company role create page', (): void => {
+    companyRoleListPage.visit();
+    companyRoleListPage.clickAddCompanyUserRoleButton();
+    companyRoleCreatePage.assertPageLocation();
+    cy.get('body').contains('Create role');
```

Comment:
avoid using page titles, translation might be changed. 
Better to check that the form is visible (form name = company_role_create_form)  || or an input from this form with id=company_role_create_form_name